addons/web/controllers/binary.py

```diff
@@ -16,11 +16,11 @@
 
 import odoo
 import odoo.modules.registry
-from odoo import SUPERUSER_ID, _, http, api
+from odoo import api, http
 from odoo.addons.base.models.assetsbundle import ANY_UNIQUE
 from odoo.exceptions import AccessError, UserError
 from odoo.http import request, Response
-from odoo.tools import file_open, file_path, replace_exceptions, str2bool
+from odoo.tools import _, file_open, file_path, replace_exceptions, str2bool
```

Comment:
is there a particular reason why the `_` import changed here, but not on any other file?

Reply:
No, I moved this one manually since I was changing the `from odoo import`.